backends/bmv2/pna_nic/main.cpp

```diff
@@ -36,6 +33,9 @@ limitations under the License.
 #include "lib/gc.h"
 #include "lib/log.h"
 #include "lib/nullstream.h"
+#include "midend.h"
```

Comment:
The other code doesn't follow this style. Any reason why you want to remove the prefix?